atomicapp/nulecule/base.py

```diff
@@ -175,8 +175,9 @@ def load_from_path(cls, src, config=None, namespace=GLOBAL_CONF,
             raise NuleculeException("Failure parsing %s file. Validation error on line %s, column %s:\n%s"
                                     % (nulecule_path, line, column, output))
 
-        nulecule = Nulecule(config=config, basepath=src,
-                            namespace=namespace, **nulecule_data)
+        nulecule = Nulecule(config=config,
+                            basepath=src, namespace=namespace,
+                            **nulecule_data)
```

Comment:
What changed here? If nothing can we put it back how it was so it doesn't show up in the diff?